packages/functions/src/sendEmail/https.ts

```diff
@@ -19,27 +26,61 @@ export const sendEmail = functions
   .region(__functionsZone__)
   .https.onCall(
     async (
-      { organization, secretKey = "", ...email }: SendEmailPayload,
-      { auth }
+      emailPayload: ClientEmailPayload[EmailType],
+      { auth }: functions.https.CallableContext
     ) => {
       if (
-        !(await checkUser(organization, auth)) &&
-        !(await checkSecretKey({ organization, secretKey }))
+        !(await checkUser(emailPayload.organization, auth)) &&
+        !(
+          emailPayload.type === EmailType.SendCalendarFile &&
+          (await checkSecretKey({
+            organization: emailPayload.organization,
+            secretKey: emailPayload.customer.secretKey,
+          }))
+        )
       ) {
         throwUnauth();
       }
 
-      checkRequiredFields(email, ["to", "html", "subject"]);
+      // get email temp from org doc
+      const emailTemplate = await fetchOrganizationEmailTemplate(
```

Comment:
Can we name this `getOrganizationTemplate` rather than `fetch` to conform to the naming we use so far.